students_documentation/summary_notebook/summary.qmd

```diff
@@ -249,6 +249,44 @@ end
 mosaic(images..., ncol=10) |> display
 ```
 
+#### UCI Adult dataset
+
+Added the UCI adult dataset as a benchmark dataset.
+
+```{julia}
+adult_data = CounterfactualExplanations.Data.load_uci_adult()
+adult_data_x = adult_data.X
+adult_data_y = adult_data.output_encoder.y
+```
+
+```{julia}
+# Train a model
+M = fit_model(adult_data, :Linear)
+
+# Factual and target labels:
+target = 1
+factual = 0
+```
+
+```{julia}
+n_individuals = 5
+
+# Choose random factual individuals
+ids = rand(findall(predict_label(M, german_credit_data) .== factual), n_individuals)
+xs = select_factual(german_credit_data, ids)
+
+generator = GenericGenerator()
+
+generated_counterfactuals_german_credit_data = generate_counterfactual(
```

Comment:
german credit here, go over all of these please

Reply:
This was very sloppy of me, I'll pay better attention next time.